cli/cmd/tls.go

```diff
@@ -109,6 +109,9 @@ func openSSLCommandFunc(command *cobra.Command, args []string) {
 			}
 		}(mod)
 		runMods++
+		if gConf.IsSingleMode && runMods == 1 {
+			break
```

Comment:
how to run gnutls module ?

Reply:
`--libssl` is only for libssl library? not gnutls ?

Reply:
did a quick test, indeed can't run gnutls

```
 ./bin/ecapture  tls --pthread=/lib64/libpthread.so.0 --gnutls=/lib64/libssl.so.1.1.1g --hex --single-mode=true
2022/06/07 22:48:15 pid info :1414631
2022/06/07 22:48:15 start to run EBPFProbeOPENSSL module
2022/06/07 22:48:15 HOOK type:2, binrayPath:/lib64/libssl.so.1.1
2022/06/07 22:48:15 libPthread so Path:/lib64/libpthread.so.0
2022/06/07 22:48:15 target all process. 
```
do you have better idea on how to achieve the goal ?

Reply:
maybe add `if gConf.IsSingleMode` in each module `Check()` to return before auto detect lib could do the job, let me know your thoughts